### PR TITLE
[PORT][8.0] product_standard_margin

### PR DESCRIPTION
**Pull request by legalsylvain**
- port product_standard_margin;
- use new api;
- apply TODO notes, due to the new V8 api;
- use new stored field list_price_vat_excl;

Thanks for your review.
Kind regards.

### CODE REVIEW

File: product_standard_margin/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Product Margin and Margin Rate',
+    'version': '2.0',
+    'author': 'Camptocamp,GRAP,Odoo Community Association (OCA)',
+    'category': 'Product',
+    'depends': [
+        'product_replenishment_cost',
```
Comment (gurneyalex) [inline product_standard_margin/__openerp__.py]: this module is not found -> broken CI

No idea where this module lives, though...

**Comment by jgrandguillaume** [comment]
Hi,

:+1: once CIS are green.

Thanks for the job !

**Comment by legalsylvain** [comment]
Now green ('product_replenishment_cost'  is now available.)
Thanks for your review.

File: product_standard_margin/README.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
+Product Margin and Margin Rate
```
Comment (yvaucher) [inline product_standard_margin/README.rst:4]: Could you add the AGPL logo ?

File: product_standard_margin/models/product_product.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+class ProductProduct(Model):
+    _inherit = 'product.product'
+
+    @api.multi
+    @api.depends(
+        'product_tmpl_id.list_price', 'replenishment_cost', 'taxes_id.type',
+        'taxes_id.price_include', 'taxes_id.amount',
+        'taxes_id.include_base_amount', 'taxes_id.child_depend')
+    def _get_margin(self):
+        tax_obj = self.pool['account.tax']
```
Comment (yvaucher) [inline product_standard_margin/models/product_product.py]: self.env ?

File: product_standard_margin/models/product_product.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        for product in self:
+            product.list_price_vat_excl = tax_obj.compute_all(
```
Comment (yvaucher) [inline product_standard_margin/models/product_product.py]: There is a `@api.v8` version of this method https://github.com/odoo/odoo/blob/dd4566de37559697af68d2239429e460c16e17d1/addons/account/account.py#L2140-L2144

File: product_standard_margin/README.rst (hunk 2 above)
Comment (legalsylvain) [inline product_standard_margin/README.rst:4]: @yvaucher : Done.

**Comment by yvaucher** [comment]
Thanks :+1:

**Pull request merged**